docs/Authentication.md

```diff
@@ -491,40 +510,40 @@ const authProvider = {
 
 React-admin calls the `authProvider` methods with the following params:
 
-| Method           | Usage                                           | Parameters format  |
-| ---------------- | ----------------------------------------------- | ------------------ |
-| `login`          | Log a user in                                   | `Object` whatever fields the login form contains |
-| `checkError`     | Check if a dataProvider error is an authentication error  | `{ message: string, status: number, body: Object }` the error returned by the `dataProvider` |
-| `checkAuth`      | Check credentials before moving to a new route  | `Object` whatever params passed to `useCheckAuth()` - empty for react-admin default routes |
-| `logout`         | Log a user out                                  |                    |
-| `getIdentity`    | Get the current user identity                   |                    | 
-| `getPermissions` | Get the current user credentials                | `Object` whatever params passed to `usePermissions()` - empty for react-admin default routes |
+| Method           | Usage                                                    | Parameters format                                                                            |
```

Comment:
please revert these table formatting changes. They make the doc harder to read when not translated into Markdown, and they don't bring any value.

Reply:
I think it's related to my prettier config, this happens every time! I'll revert